index.html

```diff
@@ -8,9 +8,9 @@
 
 <link rel="stylesheet" href="style.css">
 
```

Comment:
Kindly update the name of your feature branch to represent the general action that you performed in the branch e.g `create-home-page` 🙏. To do this, checkout to a new branch and make sure to give it a descriptive name. Then open a new pull request and ensure that you add a link to this pull request in the description 👍